devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -19,8 +19,10 @@ metadata {
         capability "Smoke Detector" //attributes: smoke ("detected","clear","tested")
         capability "Temperature Measurement" //attributes: temperature
         capability "Health Check"
-        attribute "tamper", "enum", ["detected", "clear"]
+        capability "Tamper Alert"
+		
         attribute "heatAlarm", "enum", ["overheat detected", "clear", "rapid temperature rise", "underheat detected"]
```

Comment:
I'd update this to the Temperature Alarm capability that has been defined since this was originally written.